runtime/dotnet/azurefunctions/Startup.cs

```diff
@@ -37,20 +37,14 @@ private IConfigurationRoot BuildConfiguration(string rootDirectory)
 
             config
                 .SetBasePath(rootDirectory)
-                .AddJsonFile("ComposerDialogs/settings/appsettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
```

Comment:
are you writing the settings differently?
in teh web app version, we load appsettings, then settings/appsettings, then finally appsettings.deployment.

the `settings/appsettings` version contains the settings managed inside Composer.  the root appsettings.json is only modified by hand by the user.

Reply:
Great catch! Pushed a fix for this. Still testing a bit more.